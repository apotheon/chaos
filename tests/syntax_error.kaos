void num def a()
    print "inside function a"
    num val1 = 5
    print val1
end
