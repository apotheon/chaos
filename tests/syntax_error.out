[1;46m  Syntax error:                                             [0m
[0;46m    File: "tests/syntax_error.kaos", line 1, cause: num     [0m
[0;46m      void num def a()                                      [0m
